sci-astronomy/pal/files/pal-0.9.8-fix-strlcpy-musl.patch

```diff
@@ -0,0 +1,13 @@
+Bug: https://bugs.gentoo.org/898106
+bsd/string.h is available on most libc's
```

Comment:
`string.h` should always exist (POSIX), but `bsd/string.h` is from libbsd.

The actual issue here is "does the libc's string.h provide strlcpy?"

The answer is:
* musl does under _GNU_SOURCE
* >=glibc-2.38 does under _GNU_SOURCE
* both of them will provide it in general "soon" as it's in the next edition of POSIX

Reply:
So should it be using something like
```c
#define _GNU_SOURCE
#include <string.h>
```
instead?